test/infer/test_autoguide.py

```diff
@@ -104,12 +106,12 @@ def body_fn(i, val):
     # Predictive can be instantiated from posterior samples...
     predictive = Predictive(model, posterior_samples=posterior_samples)
     predictive_samples = predictive(random.PRNGKey(1), None)
-    assert predictive_samples["obs"].shape == (1000, 2)
+    assert predictive_samples["obs"].shape == (1000, N, 2)
```

Comment:
does this mean predictive was generating incorrect samples before?

Reply:
That's right. Previously we missed the plate around the likelihood `numpyro.sample("obs", dist.Bernoulli(f), obs=data)` so if data is None, the likelihood will have the same shape as `f`, which is `(2,)`.